Components/tla/EAM/Once/Cli/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
-  "extends": [
```

Comment:
rules come now from the aslant "UcpComponent" and can be applied in each other UcpComponent